Python/Watermark_Images/README.md

```diff
@@ -0,0 +1,16 @@
+# Watermark Images
+
+## Description:
+Simple python scripts that can be used to add watermark to an image using either text or logo.
+
+## Requirements to run the notebook:
+
+>1. OpenCV
+>2. Numpy
+>3. Python Imaging Library(PIL)
```

Comment:
Make it an unordered list. It would be great if you can have this in a `requirements.txt` so that the user can just spin up a virtual environment and install it readily

Reply:
Done.